### PR TITLE
Ensure command computations set CWD relative to computation file.

### LINKED ISSUE #38: Simulator miss-sets CWD for CommandComputation
_coinstac-simulator_ doesn’t properly resolve the `cwd` for `CommandComputation`s.

### PR DESCRIPTION
**Pull request by swashcap**
This refactors pool-initializer.js and stub-computation-to-registry.js into the
new get-pool-config.js. This file now creates the DBRegistry and
ComputationRegistry instances necessary for a PipelineRunnerPool and configures
the ComputationRegistry to include the computation under simulation. It also
addresses the computation registry's path resolution issue, which fixes the
command computation's bad CWD problem.

Closes #38.

### CODE REVIEW

File: packages/coinstac-simulator/src/boot-client.js [hunk 1]
```diff
@@ -67,16 +69,6 @@ function boot({
       pool.events.on('error', logger.error);
 
       return pool.init();
-    })
-    .then(() => {
```
Comment (swashcap) [inline packages/coinstac-simulator/src/boot-client.js:71]: `getPoolConfig` takes care of all this in both the client and server processes.

File: packages/coinstac-simulator/src/boot-client.js [hunk 2]
```diff
@@ -201,7 +193,7 @@ const kickoff = function kickoff() {
 process.on('message', (opts) => {
   if (opts.boot) {
     boot(opts.boot)
-    .then((result) => process.send({ ready: true, result }));
```
Comment (swashcap) [inline packages/coinstac-simulator/src/boot-client.js:204]: `result` is now the return value of `PipelineRunnerPool#init`, which contains self-references. This causes `process.send` to crash.

File: packages/coinstac-simulator/test/coinstac-simulator.js [hunk 3]
```diff
@@ -375,3 +375,12 @@ tape('runs declaration :: errors', t => {
     });
 });
 
+tape('sets commands\' cwd', t => {
```
Comment (swashcap) [inline packages/coinstac-simulator/test/coinstac-simulator.js:378]: You betcha, we testin’.

File: packages/coinstac-simulator/test/mocks/exec-computation.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
```
Comment (swashcap) [inline packages/coinstac-simulator/test/mocks/exec-computation.js:3]: Hard to tell whether this totally covers us, but `CommandComputation`s in both `local` and `remote` should be good. Also, this test is _noisy_.

**Review by rssk — APPROVED** [review]
Looks good 💭

File: packages/coinstac-simulator/src/boot-client.js (hunk 1 above)
Comment (rssk) [reply packages/coinstac-simulator/src/boot-client.js:71]: Yo dawg, I heard you like pools....

**Comment by swashcap** [comment]
@rssk How do you feel about merging?

**Pull request merged**